robolectric-shadows/shadows-core/src/main/java/org/robolectric/shadows/ShadowActivity.java

```diff
@@ -431,6 +431,16 @@ public IntentForResult(Intent intent, int requestCode, Bundle options) {
   }
 
   @Implementation
+  public void startActivities(Intent[] intents) {
```

Comment:
Yes, that would probably avoid the need for the methods, however it would also pull in other methods from the parent classes, some of which are already in play due to the shadow system.  We thought that including them explicitly would be the smallest change.